anatomy/tracks/concept-exercises.md

```diff
@@ -72,6 +72,19 @@ exercises
         └── CarsAssemblyTests.cs (tests)
 </pre>
 
+### Minimal valid spec
+
+We favor an "optimistic merging" approach to new exercises, where tracks can develop exercises in a "work in progress" state.
+The minimal valid state, which will pass configlet and allow you to merge is:
+- Valid entry in the track `config.json`, with the `status` set to `wip`.
+- A valid `.meta/config.json` file
+- The following files being present, although they may be empty:
+  - `.docs/introduction.md`
+  - `.docs/instructions.md`
+  - `.docs/hints.md`
+  - Stub implementation
+  - Test file
+
```

Comment:
Would it perhaps also be useful to state some of the things that will _not_ be checked? Things like:

- Presence of exemplar file
- Presence of `.meta/design.md` file

Reply:
That is a really good point.  Do things "break" on the site if there is no `exemplar`?  If so, it should be checked.  And it would be nice if both the `exemplar` and the `design.md` files being absent or empty triggered a warning.

Reply:
Yeah, so
1. I think the normal configlet checks should run on a `wip` file but should be "warnings" not "errors". I can see that people might disagree with this though. I don't have opinions on confliget beyond this really. Whatever maintainers feel is best.
2. Examplar won't be used for `wip` files at launch. It is atm as its rendered with the instructions, but its absence won't break anything.
3. `design.md` isn't used by the website.

I'm happy to add a bit about the things that aren't checked, but it's another thing to keep in sync (I'm always wary of listing absent things as I think they more easily get outdated). But happy if people think it's helpful :)